src/PokeSearch.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import './PokeSearch.css';
+
+export default class PokeSearch extends Component {
+  state = { searchField: '', sortField: '' }
+
+  handleSortField = ({ target }) =>
+    this.setState({ sortField: target.value })
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const onSearch = this.props.onSearch;
+    const searchField = this.state.searchField;
+    onSearch(searchField);
```

Comment:
you'll want to pass in `this.state` to onSearch instead of just `this.state.searchField` to include that sortField state also